### PR TITLE
Create an experimental mixed_sp_wrr_traffic_test.go - part 1

### PR DESCRIPTION
**Pull request by sezhang2**
Added QoS traffic from 2 input interfaces

Test case:

- https://github.com/openconfig/featureprofiles/blob/main/feature/qos/ate_tests/mixed_sp_wrr_traffic_test/README.md

### CODE REVIEW

**Review by nraghavendran — APPROVED** [review]

File: feature/experimental/qos/ate_tests/mixed_sp_wrr_traffic_test/mixed_sp_wrr_traffic_test.go [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+		WithDefaultGateway("198.51.100.2")
+	intf3 := top.AddInterface("intf3").WithPort(ap3)
+	intf3.IPv4().
+		WithAddress("198.51.100.5/31").
+		WithDefaultGateway("198.51.100.4")
+	top.Push(t).StartProtocols(t)
+
+	var trafficFlows map[string]*trafficData
+
+	switch dut.Vendor() {
```
Comment (dplore) [inline feature/experimental/qos/ate_tests/mixed_sp_wrr_traffic_test/mixed_sp_wrr_traffic_test.go:89]: @sezhang2 why do we have more flows for ARISTA vs. Juniper and Cisco which appear to be the same except for queue assignment?   Different queue numberings per vendor seems ok?  But shouldn't the traffic on the wire be the same across all vendor cases?

Comment (sezhang2) [reply feature/experimental/qos/ate_tests/mixed_sp_wrr_traffic_test/mixed_sp_wrr_traffic_test.go:89]: Yes, the traffic would same for all vendors based on our test plan.
  - https://github.com/openconfig/featureprofiles/blob/main/feature/qos/ate_tests/mixed_sp_wrr_traffic_test/README.md

Comment (dplore) [reply feature/experimental/qos/ate_tests/mixed_sp_wrr_traffic_test/mixed_sp_wrr_traffic_test.go:89]: Ok, but the traffic flows are slightly different for arista for some reason?  trafficrate and framesize for example are not the same here:

https://github.com/openconfig/featureprofiles/pull/977/files/d6663d697873a778d75f3e2bddef48c8d9b5bc6e#diff-7ad47340a83785ccaedf86016c00d364191da968a47ee2a468cabda638549e23R92

as here:
https://github.com/openconfig/featureprofiles/pull/977/files/d6663d697873a778d75f3e2bddef48c8d9b5bc6e#diff-7ad47340a83785ccaedf86016c00d364191da968a47ee2a468cabda638549e23R107

Comment (sezhang2) [reply feature/experimental/qos/ate_tests/mixed_sp_wrr_traffic_test/mixed_sp_wrr_traffic_test.go:89]: > Ok, but the traffic flows are slightly different for arista for some reason? trafficrate and framesize for example are not the same here:
> 
> https://github.com/openconfig/featureprofiles/pull/977/files/d6663d697873a778d75f3e2bddef48c8d9b5bc6e#diff-7ad47340a83785ccaedf86016c00d364191da968a47ee2a468cabda638549e23R92
> 
> as here: https://github.com/openconfig/featureprofiles/pull/977/files/d6663d697873a778d75f3e2bddef48c8d9b5bc6e#diff-7ad47340a83785ccaedf86016c00d364191da968a47ee2a468cabda638549e23R107

Ack. The traffic flow will have same values as described in the test plan in the next commit.

**Pull request merged**